grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -1162,13 +1183,16 @@ export class MarimekkoChart
                 row.value,
             ])
         )
+        const labelCanidateSource = xColumnAtLastTimePoint
```

Comment:
continuing with me always finding a typo in your PRs:
should be `labelCandidateSource` (you're missing a `d`)